pkg/api/server/v1alpha2/results_test.go

```diff
@@ -49,7 +51,11 @@ func TestCreateResult(t *testing.T) {
 		}
 		want := proto.Clone(req.GetResult()).(*pb.Result)
 		want.Id = fmt.Sprint(lastID)
-		if diff := cmp.Diff(got, want, protocmp.Transform()); diff != "" {
+		opts := cmp.Options{
+			protocmp.Transform(),
+			protocmp.IgnoreFields(&pb.Result{}, "created_time", "updated_time"),
```

Comment:
I wouldn't ignore the fields here - we're not testing that these fields are actually set (this would be okay for watcher tests).

Reply:
That's right! Done.